caluma/workflow/validators.py

```diff
@@ -5,15 +5,15 @@
 
 
 class WorkItemValidator:
-    def _validate_task_complete_task_form(self, task, case, document):
-        DocumentValidator().validate(answers=document.answers, form=case.document.form)
-
     def _validate_task_simple(self, task, case, document):
         pass
 
     def _validate_task_complete_workflow_form(self, task, case, document):
         self._validate_task_complete_task_form(task, case, case.document)
 
+    def _validate_task_complete_task_form(self, task, case, document):
```

Comment:
I guess this change is not needed in the diff

Reply:
True, reverted.